packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/auth/FlutterAuthProvider.kt

```diff
@@ -61,16 +61,16 @@ class FlutterAuthProviders(private val methodChannel: MethodChannel) {
      * Retrieves the token for [authType] or `null`, if unavailable.
      *
      * This function is typically called from within the Amplify library and from a thread besides
-     * the main one, where it is safe to block. In API REST, the calling thread is main and we must
-     * return `null`.
-     *
-     * Not blocking the main thread is required for making platform channel calls without deadlock.
+     * the main one, where it is safe to block.
      */
     fun getToken(authType: AuthorizationType): String? {
+        // Not blocking the main thread is required for making platform channel calls without
+        // deadlock.
         if (Thread.currentThread() == Looper.getMainLooper().thread) {
-            // API REST will call this function from the main thread on configuration. This is bad.
-            // Since we have to block the calling thread to retrieve the token, just return null.
-            Log.e(tag, "REST OIDC/Lambda is not supported yet.")
+            Log.e(tag, """
```

Comment:
nit: Not sure if should follow the existing message of filing issue on GH 

https://github.com/aws-amplify/amplify-flutter/blob/1959ea13fb6287a544987021e5225576059e10d6/packages/amplify_core/ios/Classes/Support/ErrorMessages.swift#L20-L24